### PR TITLE
Contact form

### PR DESCRIPTION
**Pull request by ridwanediallo**
# Portfolio: contact form

> The template used: [Template 1](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1)

## PR cover these tasks:

- Create a contact me section according to the aforementioned template.
- Name field accepts up to 30 chars.
- Email field accepts only emails.
- Message field accepts up to 500 chars.
- All fields have placeholder text.
- All fields have the correct type.
- A user cannot submit the form if one or more of the fields is empty.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -250,99 +243,148 @@ <h2 class="tonic">
...
+            <hr>
           </div>
         </div>
       </div>
     </section>
-</main>
+
+  </main>
+  <footer>
+    <div class="footer-text">
```
Comment (Deepakdanger) [inline index.html:345]: - [x] The alignment of the background picture in the original website is a little down, Please fix it.
- Original website
original1_LI

- Your Website
original2

**Review by Deepakdanger — CHANGES_REQUESTED** [review]
Hi @ridwanediallo & @Sboursen, 

Good job so far! 👏
There are some issues that you still need to work on to go to the next project but you are almost there!


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- [x] It is advisable to update your readme by including all contributor's descriptions.
- [x]  There is a Typo error in the PR description, Your PR should be described your changes made during building this project.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (Sboursen) [reply index.html:345]: Hi @Deepakdanger,
Thank you for your review.

**Review by elmar8287 — APPROVED** [review]
Hi team , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 

Congratulations! 🎉🎉🎉 

**OPTIONAL: Please, fix your project live demo link displaying.** 


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Sboursen** [comment]
> Hi team ,
> 
> Your project is complete! There is nothing else to say other than... it's time to merge it :shipit:
> 
> Congratulations! tadatadatada
> 
> **OPTIONAL: Please, fix your project live demo link displaying.**
> 
> Cheers and Happy coding!clapclapclap
> 
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
> 
> _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

Thank you for the feedback @elmar8287 :100:

**Pull request merged**